### ISSUE TITLE
Using dask with clustering algorithms not included in dask_ml

### ISSUE THREAD

**Issue by lauren-gaiascope** [pangeo-data/pangeo#744]
I'm trying to use dask to run the sklearn's dbscan across multiple cores, but running into issues.  This is not an algorithm included in dask_ml, so I have been trying to call it as follows (similar to how it is used here: https://github.com/dask/dask-tutorial/issues/80):
```
client = Client(processes=False, n_workers=5)
    model = DBSCAN(eps=eps, min_samples=min_samps,metric=distance_sphere_and_time)
    with parallel_backend('dask'):
        model.fit(X)
```
where `distance_sphere_and_time` is a custom metric. 
Despite calling multiple workers, when I look at the CPU utilization, it still appears as though only one core is being used. Am I doing something wrong, or is there a reason dask can not increase the core usage with dbscan?

Also, I wasn't sure if there was a specific repo that this question is better suited for, but please feel free to redirect me if there is one.

**Comment by TomAugspurger** [comment]
The dask-ml issue tracker is probably a better spot, but no worries.

Do things work if you pass n_jobs=-1 do the DBSCAN constructor?

**Comment by lauren-gaiascope** [comment]
Thanks for the suggestion, Tom - I just gave that a try, and my cpu usage is still hovering right around 2%.

**Comment by lauren-gaiascope** [comment]
I guess I might have asked this prematurely. I just tried including `n_jobs` in the DBSCAN constructor and removing the inputs in the `Client` constructor and it appears to be working (shown below for completeness)

```
client = Client()
    model = DBSCAN(eps=eps, min_samples=min_samps,metric=distance_sphere_and_time, n_jobs=-1)
    with parallel_backend('dask'):
        model.fit(Distance)
```

It's at least using more of the cores (not all of them yet) and the CPU usage has risen to ~14%

**Comment by TomAugspurger** [comment]
You should also see some activity on the Dask dashboard if it's really
using the cluster.

**Comment by RichardScottOZ** [comment]
Did this end up working @lauren-gaiascope ?  Thanks.
